Source/WebInspectorUI/UserInterface/Models/Cookie.js

```diff
@@ -155,6 +159,9 @@ WI.Cookie = class Cookie
         let domain = null;
         let secure = false;
         let httpOnly = false;
+        // partition key is implicit and is the first party site if the Partitioned attribute is present.
+        const partitionKey = null;
```

Comment:
rather than have it be `null` i feel like we should set it equal to the first party site instead of having this be implicit